app/src/main/assets/html/error/secure-connection-failed.html

```diff
@@ -0,0 +1,22 @@
+<!doctype html>
+<html lang="en">
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="css/styles.css">
+    <title>Secured Connection Failed</title>
+    <body>
```

Comment:
should this have a `data-type` like [elsewhere](https://github.com/MozillaReality/FirefoxReality/pull/170/files#diff-42f78899dec3c3554a58e0b44c9c7a46R6)?